t/crud_spec.t

```diff
@@ -48,6 +52,12 @@ for my $dir ( map { path("t/data/CRUD/$_") } qw/read write/ ) {
         my $name = $path->relative($dir)->basename(".json");
 
         subtest $name => sub {
+            if ( $name =~ 'arrayFilter' ) {
+                my $ver = $feature_comp->output->{featureCompatibilityVersion};
+                $ver = $ver->{version} if ref($ver) eq 'HASH';
```

Comment:
Please pull the "get a version from featureCompatibilityVersion" logic into a helper sub or else just up to the preamble after line 41.  I'd like to see the `plan skip_all` predicate below simplified to something like `unless $supports_array_filters`.

Reply:
Yep will do! was debating whether to do that but forgot to come back to it after it was working heh